pkg/secrethub/service.go

```diff
@@ -8,13 +8,15 @@ import (
 // ServiceService handles operations on service accounts from SecretHub.
 type ServiceService interface {
 	// Create creates a new service account for the given repo.
-	Create(path string, description string, credential Credential, encrypter Encrypter) (*api.Service, error)
+	Create(path string, description string, verifier Verifier, encrypter Encrypter) (*api.Service, error)
```

Comment:
In practice, what does a call to this function look like?

```go
// RSA
client.Services().Create("company/repo", "my service", credential, credential)
// AWS
client.Services().AWS().Create(path, description, keyID, role, cfgs...)
```

You already have a separate function for AWS, so for the RSA function why not combine `verifier` and `encrypter` in the same argument with a joined interface?

Reply:
Also, how do you feel about having the interface composed this way with a separate AWS interface but having the `Create` function on the 'root' `ServiceService` interface?

Reply:
In hindsight, splitting `verifier` and `encrypter` does indeed seem to be unnecessary.

Reply:
> Also, how do you feel about having the interface composed this way with a separate AWS interface but having the Create function on the 'root' ServiceService interface?

What do you mean?

Reply:
Wow hard to read question I formulated there. Wha tI meant is we have a 'plain` service that will be created on `client.Services().Create()` and an AWS service that will be created on a level of nesting deeper: `client.Services().AWS().Create()`. 

My question is, should we then make 'plain' services be created on e.g. (and this is far from pretty) `client.Services().Default().Create()`?